### ISSUE TITLE
fullscreen button & iframes

### ISSUE THREAD

**Issue by seyfro** [Leaflet/Leaflet.fullscreen#9]
a user of my plugin reported that the fullscreen button does not work if the map is loaded within an iframe. Is there anything that can be done about this?
thx,
Robert

**Comment by jfirebaugh** [comment]
Can you provide a test case?

**Comment by seyfro** [comment]
Sorry, of course - here´s a quick test case: http://current.mapsmarker.com/2013/09/22/iframe-fullscreen-not-working/

**Comment by connor** [comment]
I'll leaveit up to @jfirebaugh to determine if it's within this library's goals to accommodate iframe fullscreen requests, but I just put together a demo using [bigscreen.js](https://github.com/bdougherty/BigScreen) and replaced all of the `requestFullscreen` and `exitFullscreen` calls with `BigScreen.toggle(container)`, and that's working great for me.

@robertharm if you need support for this immediately, that looks like a possible solution.

**Comment by seyfro** [comment]
thanks for the info!

**Comment by jfirebaugh** [comment]
You need to set [`allowfullscreen="true"`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen) (and for maximum backward compatibility `mozallowfullscreen=true` / `webkitallowfullscreen=true`) on the iframe element to allow the iframe content to request fullscreen.

**Comment by mtbsomi** [comment]
Hi!

It's 2016, but I still have problem with fullscreen in an iframe.

Embed code:

<iframe id="id" src="src" frameborder="0" style="width:100%;height:100%;min-height:400px;border:0" allowfullscreen></iframe>

Output in fullscreen and in normal mode:

http://imgur.com/a/2iA4J

Sorry for bluring out almost eveything. It's a page which is in development status. However the blur it is clear there's no problem in normal mode, and the fullscreen mode as a mode is actually works, but in visualization there's a problem.

I use this iframe techniqe and this addon, and there's no problem, so I think there will be some interesting CSS related problem here.

[...] 

I've made a final test, and I got it!

It's really a CSS issue. The panel on right side is animated (using Animate.css - http://daneden.me/animate) and there is a line for .animated:

animation-fill-mode: both;

This makes the problem.

[...]

https://github.com/matthieua/WOW/issues/115

Same prob here, no solution with both animation and iframe fullscreen.

I decided to leave my comment here as a memo. Maybe you have better ideas then the other guys?! :)

Best wishes.

UPDATE: animations are working without that line, and there's fullscreen mode now. Interesting. Where's the poop? :)
